misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/NodeIndentation.java

```diff
@@ -111,8 +112,20 @@ static int simpleNameReferenceNode(Node node, FormattingOptions options) {
             }
         }
         if (parent != null && parent.kind() == SyntaxKind.FUNCTION_CALL) {
+            if (grandParent != null && grandParent.kind() == SyntaxKind.CALL_STATEMENT) {
```

Comment:
can simplify the addSpaces value assignment by assigning the if condition directly

Reply:
Will make the following improvement in a refactoring PR.
`addSpaces = grandParent != null && grandParent.kind() == SyntaxKind.CALL_STATEMENT;`